reactions.json

```diff
@@ -163,12 +166,22 @@
     "HgSO4 / H2SO4": "propanone"
   },
   "methanal": {
-    "HCN": "formaldehyde cyanohydrin",
-    "NaHSO3": "formaldehyde sodium bisulphite"
+    "HCN": "hydroxyethanenitrile",
+    "_NaHSO3": "hydroxymethanesulfonic acid",

```

Comment:
Why is there an underscore? also the name is still not correct

Reply:
the underscore is there because I ain't sure about the name